hack/operatorhub/README.md

```diff
@@ -1,23 +1,224 @@
+./operatorhub container publish -a 'api-key-in-keybase' -p `project-id` -t 2.6.0 -r `registry-password-for-quay.io` --dry-run=false
+```
+
+Usage with vault
+```shell
+OHUB_TAG=2.6.0-bc2 OHUB_GITHUB_VAULT_SECRET="secret/ci/elastic-cloud-on-k8s/operatorhub-release-github" OHUB_REDHAT_VAULT_SECRET="secret/ci/elastic-cloud-on-k8s/operatorhub-release-redhat" VAULT_ADDR='https://vault-server:8200' VAULT_TOKEN=my-token ./bin/operatorhub container publish --enable-vault --dry-run=false
+```
+
+### Flags
+
+| Parameter              | Description                                                                                                                               | Environment Variable     | Default |
+|----------------------  |-------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|---------|
+| `--api-key`            | API key to use when communicating with redhat certification API.                                                                          | `OHUB_API_KEY`           | `""`    |
+| `--registry-password`  | Registry password used to communicate with Quay.io.                                                                                       | `OHUB_REGISTRY_PASSWORD` | `""`    |
+| `--project-id`         | Red Hat project id within the Red Hat technology portal.                                                                                  | `OHUB_PROJECT_ID`        | `""`    |
+| `--tag`                | Git tag to release.                                                                                                                       | `OHUB_TAG`               | `""`    |
+| `--dry-run`            | Only validation will be run for the push command if set. The publish command will ensure the image is scanned but will not publish if set.| `OHUB_DRY_RUN`           | `true`  |
+
+## Generate Manifests Command
+
+The `generate-manifests` command will extracts CRDs and RBAC definitions from distribution YAML manifests
+(either yaml manifests from flag, or pulled from internet) and generates the files required to publish
+a new release to OperatorHub.  These files are written within the following 3 directories:
+
+- community-operators
+- upstream-community-operators
+- certified-operators
+
+### Usage
+
+To generate configuration for a previously released manifest version
 
 ```shell
-go run main.go --conf=config.yaml 
+./bin/operatorhub generate-manifests -c config.yaml -t '2.5.0'
 ```
 
 To generate configuration based on yet unreleased YAML manifests:
 
 ```shell
-go run main.go --conf=config.yaml --yaml-manifest=../../config/crds.yaml --yaml-manifest=../../config/operator.yaml
+./bin/operatorhub generate-manifests -c config.yaml -y ../../config/crds.yaml -y ./../config/operator.yaml -s '8.6.0' -V '2.5.0' -t '2.6.0'
```

Comment:
Why asked `-s '8.6.0' -V '2.5.0' -t '2.6.0'`when it's already defined in `-c config.yaml`?

Reply:
yeah, I will get this simplified.  good point.  Unnecessary.